CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## Version 2.8.0-beta3
```

Comment:
please advise what is our plan for beta3 and whether i need to change it to 2.9-beta1

Reply:
there was an informal discussion about a beta3, but i think the plan is to go on to 2.9-beta1.
Lets get this cleared up during standup today.